libraries/AP_Notify/NeoPixel.cpp

```diff
@@ -20,9 +20,7 @@
 // This limit is from the dshot driver rcout groups limit
 #define AP_NOTIFY_NEOPIXEL_MAX_INSTANCES        4
 
-#ifndef HAL_NEOPIXEL_COUNT
-#define HAL_NEOPIXEL_COUNT 1
-#endif
+#define HAL_NEOPIXEL_COUNT (pNotify->get_neo_len()) //uses param NTF_NEO_LEN
```

Comment:
I think its easier to understand to get rid of the #define and grab the param where its used